### PR TITLE
Jenkins: First tsa environment

### PR DESCRIPTION
**Pull request by havogt**

### CODE REVIEW

File: jenkins/envs/tsa.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+export GTRUN_SBATCH_NODES=1
+export GTRUN_SBATCH_NTASKS_PER_CORE=1
+export GTRUN_SBATCH_NTASKS_PER_NODE=1
+#export GTRUN_SBATCH_CPUS_PER_TASK=24
+export GTRUN_SBATCH_GRES='gpu:1'
+export GTRUNMPI_SBATCH_PARTITION='debug'
+export GTRUNMPI_SBATCH_NODES=4
+
+#export CUDA_AUTO_BOOST=0
+#export GCLOCK=1328
```
Comment (fthaler) [inline jenkins/envs/tsa.sh:28]: For Tsa the value would probably be 1530 (according to nvidia-smi), disabling auto boost might still make sense for benchmark reproducibility.

File: jenkins/envs/tsa.sh [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+#export GTRUN_BUILD_COMMAND='srun -C gpu -p cscsci --time=00:20:00 make -j 24'
+export GTRUN_BUILD_COMMAND='make -j 8'
+export GTRUN_SBATCH_PARTITION='debug'
+export GTRUN_SBATCH_NODES=1
+export GTRUN_SBATCH_NTASKS_PER_CORE=1
+export GTRUN_SBATCH_NTASKS_PER_NODE=1
+#export GTRUN_SBATCH_CPUS_PER_TASK=24
+export GTRUN_SBATCH_GRES='gpu:1'
+export GTRUNMPI_SBATCH_PARTITION='debug'
```
Comment (fthaler) [inline jenkins/envs/tsa.sh:24]: Just to make sure: have you tried the MPI tests?

File: jenkins/envs/tsa.sh [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+export GTRUNMPI_SBATCH_NODES=4
+
+#export CUDA_AUTO_BOOST=0
+#export GCLOCK=1328
+#export MPICH_RDMA_ENABLED_CUDA=1
+#export MPICH_G2G_PIPELINE=30
```
Comment (fthaler) [inline jenkins/envs/tsa.sh:30]: Why are those flags not needed (at least the first one)? Is Tsa not using Cray-MPICH?

File: jenkins/envs/tsa.sh [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+#export OMP_NUM_THREADS=24
```
Comment (fthaler) [inline jenkins/envs/tsa.sh:31]: Should/could be set to 16 I guess.

File: jenkins/envs/tsa.sh (hunk 3 above)
Comment (havogt) [reply jenkins/envs/tsa.sh:30]: They use openmpi

File: jenkins/envs/tsa.sh (hunk 2 above)
Comment (havogt) [reply jenkins/envs/tsa.sh:24]: I think I didn't run them. Let me try again.

**Comment by havogt** [comment]
launch jenkins

**Comment by havogt** [comment]
launch jenkins

**Comment by havogt** [comment]
launch jenkins

**Comment by havogt** [comment]
launch jenkins
